kiosk/forms.py

```diff
@@ -1,5 +1,29 @@
 from django import forms
 
+from core.models.DepartmentModels import Department
```

Comment:
Way to go, Haaaavard

Reply:
Congratz on getting a computer with an OS with a browser that supports btn presses 🎉